int32 index
int32 mode
int32 direction
int32 target_position
float32 target_velocity
float32 p_gain
float32 i_gain
float32 d_gain
float32 acc_ff
float32 fri_ff
float32 vel_ff
float32 hold
float32 bias
float32 accel
float32 trans
float32 lag
float32 min
float32 max
float32 slew
---
int32 ack
